src/main/java/WayofTime/bloodmagic/util/handler/event/GenericHandler.java

```diff
@@ -405,6 +410,15 @@ public static void onLivingDrops(LivingDropsEvent event) {
         }
     }
 
+    @SubscribeEvent
+    public static void onRitualDeath(LivingDropsEvent event){
+        if(!ConfigHandler.values.wellOfSufferingDrops){
+            if(event.getSource().equals(WayofTime.bloodmagic.ritual.RitualManager.RITUAL_DAMAGE)) {
```

Comment:
Why fully qualified class name?

Reply:
Because you were like "static import bah" and I was like "okay, so I'll not go for an import at all because it's only needed once" and went for the fully qualified class name.